slurp/src/test/utils.test.js

```diff
@@ -0,0 +1,19 @@
+const utils = require('../main/webapp/scripts/utils.js');
+
+test('single parse no special chars', () => {
+  const url = "test.html?key1=value1";
+  const urlParams = utils.parseUrl(url);
+  expect(urlParams).toEqual({"key1": "value1"});
+});
+
+test('multiple parse no special chars', () => {
+  const url = "test.html?key1=value1&key2=value2";
+  const urlParams = utils.parseUrl(url);
+  expect(urlParams).toEqual({"key1": "value1", "key2": "value2"});
+});
+
+test('multiple parse with special chars', () => {
+  const url = "test.html?key1=!#sJk3r2Iedy2dw[-cc2>&key2=value2";
+  const urlParams = utils.parseUrl(url);
+  expect(urlParams).toEqual({"key1": "!#sJk3r2Iedy2dw[-cc2>", "key2": "value2"});
+});
```

Comment:
This file tests utils.js, but you should also test the code in activities.js. Could you add tests for that file as well?

Reply:
All the code in activities.js in this PR is HTML-generating. Are tests necessary?